integrations/opentelemetry/src/main/kotlin/com/xebia/functional/xef/opentelemetry/OpenTelemetryConfig.kt

```diff
@@ -0,0 +1,50 @@
+package com.xebia.functional.xef.opentelemetry
+
+import io.opentelemetry.api.OpenTelemetry
+import io.opentelemetry.api.common.AttributeKey
+import io.opentelemetry.api.common.Attributes
+import io.opentelemetry.exporter.otlp.trace.OtlpGrpcSpanExporter
+import io.opentelemetry.sdk.OpenTelemetrySdk
+import io.opentelemetry.sdk.resources.Resource
+import io.opentelemetry.sdk.trace.SdkTracerProvider
+import io.opentelemetry.sdk.trace.export.BatchSpanProcessor
+import java.util.concurrent.TimeUnit
+
+data class OpenTelemetryConfig(
+    val endpointConfig: String,
+    val defaultScopeName: String,
+    val serviceName: String
+) {
+
+    fun newInstance(): OpenTelemetry {
+        val jaegerOtlpExporter: OtlpGrpcSpanExporter = OtlpGrpcSpanExporter.builder()
+            .setEndpoint(endpointConfig)
+            .setTimeout(30, TimeUnit.SECONDS)
```

Comment:
Should be this parameter taken from config or environment?

Reply:
I think that it's not necessary for now